src/Editor/GUI/Editors/EntityEditor.hpp

```diff
@@ -73,6 +74,42 @@ namespace GUI {
              */
             void SetVisible(bool visible);
             
+            /// Sets the vertex and index data.
+            /**
+            * @param the data which to set the information from.
+            */
+            void SetVertexData(Geometry::AssetFileHandler::MeshData* data);
+
+            ///Returns the vertex data.
+            /**
+            * @return The array of vertices previously loaded by the SetVertexData function.
+            */
+            Video::Geometry::VertexType::StaticVertex* GetVertices();
+
+            ///Returns the index data.
+            /**
+            * @return The array of indices previously loaded by the SetVertexData function.
+            */
+            uint32_t* GetIndices();
```

Comment:
Method could be const. The return value should probably also be const unless it is to be modified.